CMakeLists.txt

```diff
@@ -33,6 +33,17 @@ add_library(SVfit STATIC ${SVFIT_SRC})
 target_compile_definitions(SVfit PRIVATE USE_SVFITTF=1)
 set_source_files_properties(${SVFIT_SRC} PROPERTIES COMPILE_FLAGS "-w")
 
+include("${AnalysisTools_DIR}/cmake/include/tensorflow.cmake")
+file(GLOB_RECURSE MulticlassInference_SRC "${CMSSW_BASE_SRC}/MulticlassInference/MulticlassInference/src/*.cc")
+add_library(MulticlassInference STATIC ${MulticlassInference_SRC})
+target_link_libraries(MulticlassInference ${TF_LIBRARIES})
+target_include_directories(MulticlassInference PRIVATE ${TF_INCLUDES})
+set_source_files_properties(${MulticlassInference_SRC} PROPERTIES COMPILE_FLAGS "-w")
+
+target_link_libraries(CalcMulticlassDNN ${TF_LIBRARIES})
+target_include_directories(CalcMulticlassDNN PRIVATE ${TF_INCLUDES})
+set_source_files_properties("${CMSSW_BASE_SRC}/hh-bbtautau/Analysis/source/CalcMulticlassDNN.cxx" PROPERTIES COMPILE_FLAGS "-w")
```

Comment:
Is disabling warning really necessary? We already disabled all non-relevant warning here https://github.com/hh-italian-group/AnalysisTools/blob/master/cmake/include/common.cmake#L32-L37
So in general, remaining warnings are better to be solved. But of course, I'm open for discussion and some exceptions can be made.

Reply:
If they are not disabled, all warnings (couple hundred) originating from Eigen and TensorFlow code will show up here as well, so I thought leaving them enabled wouldn't be a good idea :/

Reply:
Hm... For HH-btag we needed to include only `PhysicsTools/TensorFlow/interface/TensorFlow.h` in the interface file. And the number of TF-related warnings when compiling CacheTupleProducer was rather few: about 2-4 in total, definitely not hundreds...

https://github.com/hh-italian-group/HHbtag/blob/master/interface/HH_BTag.h#L3

Reply:
This is indeed odd. I just checked again and my terminal is floated with warnings from Eigen.